index.js

```diff
@@ -89,11 +90,29 @@ const getCompiledFileSync = (moduleDir, path) => {
   return transformSrc(moduleDir, res.toString())
 }
 
+const getBsConfigModuleOptions = buildDir => {
+  const bsconfig = readBsConfigSync(buildDir)
+  if (!bsconfig) {
+    throw new Error(`bsconfig not found in ${buildDir}`)
+  }
+
+  if (!bsconfig['package-specs'] || !bsconfig['package-specs'].length) {
```

Comment:
I don't think `package-specs` needs to be defined, doesn't it default to using CommonJS?

Reply:
Oh that's right. If it's not defined i'll default to `'js'` like we do in the options part 👍